frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -983,6 +984,7 @@ pub mod pallet {
 
 			Self::deposit_event(Event::SolutionStored {
 				compute: ElectionCompute::Emergency,
+				account_id: None,
```

Comment:
May be for consistency, any signed solution should have an account id including ForceOrigin?